module/actor/actor.js

```diff
@@ -34,4 +34,63 @@ export class VampireActor extends Actor {
   //         ability.mod = Math.floor((ability.value - 10) / 2);
   //     }
   // }
+
+
+
+   /**
+   * Redefines the create "actor" type with translations :)
+   * @param {object} data         Initial data with which to populate the creation form
+   * @param {object} [options]    Positioning and sizing options for the resulting dialog
+   * @return {Promise<Document>}  A Promise which resolves to the created Document
+   * @memberof ClientDocumentMixin
+   */
+     static async createDialog(data={}, options={}) {
```

Comment:
Why do I recreate the createDialog box ?

Two reasons ? 
1. I have replaced the "character" actor type with a more clear "vampire" type. We can't delete yet the old character type but still we should'nt create new "character".

2. Translated terms, yay !